app/index.html

```diff
@@ -399,10 +402,10 @@ <h3>Please open a Workflow</h3>
                 <div class="modal-content">
                     <div class="modal-header">
                         <button type="button" class="close" data-dismiss="modal" aria-hidden="true">×</button>
-                        <h3>Publish current Workflow to Catalog</h3>
+                        <h3>Publish to Catalog</h3>
                     </div>
                     <div class="modal-body" style="height:100%">
-                        Do you want to publish your Workflow into the Catalog ?
+                        Do you want to publish your object into the Catalog ?
```

Comment:
`in the Catalog` and not `into the Catalog`
source : https://english.stackexchange.com/questions/94584/publish-in-vs-publish-on